suse/distributionscanner.go

```diff
@@ -29,38 +32,12 @@ const (
 	suseReleasePath = `etc/SuSE-release`
```

Comment:
are these used anywhere? sorry just noticed this

Reply:
yeah, they are the files we check for during the indexing